### ISSUE TITLE
Real time

### ISSUE THREAD

**Issue by trippietrae** [iGBAEmu/EclipseIssues#98]
please add real time option

**Comment by magnetardev** [comment]
Are you talking about the clock feature in some GBA games (i.e. Pokémon Emerald)?

**Comment by trippietrae** [comment]
Yes, I am playing Pokemon Glazed and I need the clock in order to get an item.
